INFO - 2025-06-14 17:17:28,462 - conftest.py:21 -[conftest:start_test_and_end] - -------------接口测试开始--------------
ERROR - 2025-06-14 17:17:28,462 - readyaml.py:115 -[readyaml:get_extract_yaml] - 【extract.yaml】没有找到：token,--'token'
ERROR - 2025-06-14 17:17:28,471 - readyaml.py:115 -[readyaml:get_extract_yaml] - 【extract.yaml】没有找到：token,--'token'
INFO - 2025-06-14 17:17:28,476 - sendrequest.py:142 -[sendrequest:run_main] - 接口名称：4更新课程状态（上架/下架）
INFO - 2025-06-14 17:17:28,476 - sendrequest.py:143 -[sendrequest:run_main] - 请求地址：http://47.113.145.175:8080/ssm_web/course/updateCourseStatus
INFO - 2025-06-14 17:17:28,476 - sendrequest.py:144 -[sendrequest:run_main] - 请求方式：get
INFO - 2025-06-14 17:17:28,476 - sendrequest.py:145 -[sendrequest:run_main] - 测试用例名称：4更新课程状态（上架/下架）
INFO - 2025-06-14 17:17:28,476 - sendrequest.py:146 -[sendrequest:run_main] - 请求头：{'Content-Type': 'application/json', 'Authorization': 'None', 'Cookie': 'JSESSIONID=AF7CCFDA4D900B484D6407ED95C46A64'}
INFO - 2025-06-14 17:17:28,477 - sendrequest.py:157 -[sendrequest:run_main] - 请求参数：{'params': {'status': 1, 'id': 289}}
INFO - 2025-06-14 17:17:28,671 - sendrequest.py:108 -[sendrequest:send_request] - 接口返回信息：{"success":true,"state":200,"message":"响应成功","content":{"status":1}}
INFO - 2025-06-14 17:17:28,683 - assertions.py:308 -[assertions:assert_result] - yaml文件预期结果：[{'contains': {'status_code': 200}}]
ERROR - 2025-06-14 17:17:28,683 - assertions.py:41 -[assertions:contains_assert] - 接口返回码包含断言成功：预期结果【200】,实际结果【200】
INFO - 2025-06-14 17:17:28,683 - assertions.py:339 -[assertions:assert_result] - 测试成功
INFO - 2025-06-14 17:17:28,683 - conftest.py:23 -[conftest:start_test_and_end] - -------------接口测试结束--------------
INFO - 2025-06-14 17:17:28,683 - db_management.py:61 -[db_management:restore_data] - 执行MySQL还原命令: mysql -u root -pLagou@1234 -P 3306 -h 47.113.145.175 ssm_lagou_edu < C:\tocode\Project\cursor\Test_API_Framework-eduProject\backup\mysql_backup_20250614_171717.sql
mysql: [Warning] Using a password on the command line interface can be insecure.
INFO - 2025-06-14 17:17:31,173 - test_course_scenario.py:37 -[test_course_scenario:setup_teardown] - 数据库还原成功
